go/database/mpt/live_trie_test.go

```diff
@@ -960,3 +960,34 @@ func BenchmarkValueInsertionInFileTrie(b *testing.B) {
 		})
 	}
 }
+
+func BenchmarkLiveTrie_WithHighNumberOfDirtyNodes_Flush(b *testing.B) {
+	// running this test for all configurations at once takes too much time
+	// hence we use only the latest one
+	config := S5ArchiveConfig
+	// largest nodes have ~512 bytes - we need to create a cache with 18 GB cache,
+	// hence we need ~ 18*2^30/2^9 = 18*2^21 ~ 38 million
+	cacheSize := 38_000_000
```

Comment:
As discussed offline, the problem size should be considerably reduced to make this a reasonable benchmark.

Reply:
Unfortunately, I wasnt able to push this test under 30s.